packages/console/src/containers/AppContent/components/Topbar/TenantSelector/components/CreateTenantModal/CreateForm.tsx

```diff
@@ -0,0 +1,107 @@
+import type { AdminConsoleKey } from '@logto/phrases';
+import { type TenantInfo, type CreateTenant, TenantTag } from '@logto/schemas';
+import { Controller, FormProvider, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import Close from '@/assets/images/close.svg';
+import HeaderIcon from '@/assets/images/create-tenant-header.svg';
+import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import Button from '@/components/Button';
+import Card from '@/components/Card';
+import CardTitle from '@/components/CardTitle';
+import FormField from '@/components/FormField';
+import IconButton from '@/components/IconButton';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import TextInput from '@/components/TextInput';
+
+import * as styles from './index.module.scss';
+
+const tagOptions: Array<{ title: AdminConsoleKey; value: TenantTag }> = [
```

Comment:
Maybe in a const file, @xiaoyijun is a master of this.

Reply:
It's OK for me, since the overall size of this file is still manageable